cif_core.dic

```diff
@@ -19286,6 +19286,593 @@ save_ATOM
+      _description_example.case
+         'Fe2 O3'
+         'Li'
+         'Si O2'
+
+    _method.purpose               Evaluation
+    _method.expression
+;
+    _atom_analytical.analyte = dREL to extract analyte symbol
+;
+
+save_
+
+save_atom_analytical.chemical_species_mass_percent
+
+    _definition.id
+        '_atom_analytical.chemical_species_mass_percent'
+    _definition.update            2022-11-19
+    _description.text
+;
+    Mass percentage of the chemical species given in
+    _atom_analytical.chemical_species as derived from elemental analysis.
+
+    This is most often used in elemental compositions determined by XRF,
+    where elements are reported as equivalent mass percentages of their
+    relevant oxide.
+
+    eg: Al is reported as Al2O3, P is reported as P2O5.
+
+;
+    _name.category_id             atom_analytical
+    _name.object_id               chemical_species_mass_percent
+    _type.purpose                 Measurand
+    _type.source                  Derived
+    _type.container               Single
+    _type.contents                Real
+    _enumeration.range            0.0:100.
+    _units.code                   none
+
+save_
+
+save_atom_analytical.chemical_species_mass_percent_su
+
+    _definition.id
+        '_atom_analytical.chemical_species_mass_percent_su'
+    _definition.update            2022-11-19
+    _description.text
+;
+    Standard uncertainty of _atom_analytical.chemical_species_mass_percent.
+;
+    _name.category_id             atom_analytical
+    _name.object_id               chemical_species_mass_percent_su
+    _name.linked_item_id
+        '_atom_analytical.chemical_species_mass_percent'
+    _units.code                   none
+
+    _import.get                   [{'file':templ_attr.cif  'save':general_su}]
+
+save_
+
+save_atom_analytical.id
+
+    _definition.id                '_atom_analytical.id'
+    _definition.update            2022-11-18
+    _description.text
+
+;
+    Arbitrary label uniquely identifying a single composition value.
+;
+    _name.category_id             atom_analytical
+    _name.object_id               id
+    _type.purpose                 Key
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Code
+
+save_
+
+save_atom_analytical.meas_id
+
+    _definition.id                '_atom_analytical.meas_id'
+    _definition.update            2022-11-18
+    _description.text
+
+;
+    Arbitrary label identifying the source of an elemental composition.
+    This code must match the _atom_analytical_source.id of the associated
+    technique in the analytical source list.
+;
+    _name.category_id             atom_analytical
+    _name.object_id               meas_id
+    _name.linked_item_id          '_atom_analytical_source.id'
+    _type.purpose                 Link
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Code
```

Comment:
Same issue as before -- `Code` is a good choice for the content type of an ID since it is case insensitive ("a" == "A'). Maybe `Word` would be better?

Reply:
`Word` it is